src/modules/ethereum/transactions/gasPrices/requests.ts

```diff
@@ -53,93 +14,35 @@ export async function fetchGasPriceFromNode(
     throw new Error(`RPC URL is missing from network config for ${chain}`);
   }
 
-  const requestDate = new Date();
   const { result } = await sendRpcRequest<string>(url, {
     method: 'eth_gasPrice',
     params: null,
   });
   const gasPrice = ethers.BigNumber.from(result).toNumber();
   return {
-    info: {
-      classic: {
-        fast: gasPrice,
-        standard: gasPrice,
-        slow: gasPrice,
-        rapid: null,
-      },
+    average: {
+      classic: gasPrice,
+      eip1559: null,
+      optimistic: null,
+      eta: null,
+    },
+    fast: {
+      classic: gasPrice,
+      eip1559: null,
+      optimistic: null,
+      eta: null,
     },
-    datetime: requestDate.toString(),
-    source: url,
   };
 }
 
-class GasChainPricesSubscription {
-  latestValue: Payload | null = null;
-  initialPromise: Promise<Payload> | null = null;
-  unsubscribe: (() => void) | null = null;
-
-  async get() {
-    if (SLOW_MODE) {
-      await wait(4000);
-    }
-    if (this.latestValue) {
-      return Promise.resolve(this.latestValue);
-    } else if (this.initialPromise) {
-      return this.initialPromise;
-    } else {
-      return Promise.race([
-        this.initiateRequest(),
-        rejectAfterDelay(10000, 'GasChainPricesSubscription'),
-      ]).catch((error) => {
-        this.initialPromise = null; // reset promise so that subsequent get() call will retry
-        throw error;
-      });
-    }
-  }
-
-  initiateRequest() {
-    this.initialPromise = new Promise((resolve) => {
-      const { unsubscribe } = client.cachedSubscribe<
-        Payload,
-        typeof namespace,
-        typeof scope
-      >({
-        namespace,
-        body: {
-          scope: [scope],
-          payload: {},
-        },
-        onData: ({ value }) => {
-          if (!value) {
-            return;
-          }
-          if (!this.latestValue) {
-            resolve(value);
-          }
-          this.latestValue = value;
-        },
-        mergeStrategy: mergeSingleEntity,
-      });
-      this.unsubscribe = unsubscribe;
-    });
-    return this.initialPromise;
-  }
-}
-
-export const gasChainPricesSubscription = new GasChainPricesSubscription();
-
 export async function fetchGasPrice(chain: Chain, networks: Networks) {
   try {
-    if (networks.supports('sending', chain)) {
-      const gasPrices = await gasChainPricesSubscription.get();
-      const chainGasPrices = gasPrices[chain.toString()];
-      if (chainGasPrices) {
-        return chainGasPrices;
-      } else {
-        throw new Error('unable to get gas prices from api');
-      }
+    const response = await ZerionAPI.getGasPrices({ chain });
```

Comment:
We used to update the value no more often than the socket api returned new data.
Now we're making new request each time, right?
Let's maybe use `queryClient` to fetch no more often than 5-10 seconds?

Reply:
Hmmm
Let me think about it.

Reply:
This is true. Let me add cache here. 
This will make a difference only for the case when we add gas price info for transactions without gas price info. Cause there we are doing 2 requests in a row.

Reply:
Or may be not. Let's discuss it